docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/use-windows-clusters/use-windows-clusters.md

```diff
@@ -70,7 +66,7 @@ Rancher will not provision the node if the node does not meet these requirements
 
 Before provisioning a new cluster, be sure that you have already installed Rancher on a device that accepts inbound network traffic. This is required in order for the cluster nodes to communicate with Rancher. If you have not already installed Rancher, please refer to the [installation documentation](../../../../getting-started/installation-and-upgrade/installation-and-upgrade.md) before proceeding with this guide.
 
-Rancher only supports Windows using Flannel as the network provider.
+Rancher supports Windows using Calico or Flannel as the network provider.
 
 There are two network options: [**Host Gateway (L2bridge)**](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#host-gw) and [**VXLAN (Overlay)**](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan). The default option is **VXLAN (Overlay)** mode.
```

Comment:
Since [line 188](https://github.com/rancher/rancher-docs/pull/1306/files#diff-1166c01078611b4717dd6490197b417b7ea01caa942801e80e93c1219c55d9b3L188) is getting dropped, are these descriptions about the two network options still relevant?

Reply:
Verified this with support as well and they are not relevant moving forward, went ahead and removed them. Thanks for the review Billy!